packages/frontend/amp/components/TopMetaExtras.tsx

```diff
@@ -6,6 +6,7 @@ import { palette } from '@guardian/pasteup/palette';
 import { pillarMap, pillarPalette } from '@frontend/lib/pillars';
 import { textSans } from '@guardian/pasteup/typography';
 import TwitterIcon from '@guardian/pasteup/icons/twitter.svg';
+import { composeLabsCSS } from '@root/packages/frontend/amp/lib/compose-labs-css';
```

Comment:
Minor, @root -> @frontend/amp...

Reply:
Would still like to look into catching this with linter rule. On the todo list!